frontend/app/enrollment/new/components/care-plan-conditions-selector.tsx

```diff
@@ -0,0 +1,40 @@
+import { Label } from '@/components/ui/label'
+import MultipleSelector from '@/components/ui/multipleselect'
+import useEnrollment from '@/lib/store/enrollment-store'
+import React from 'react'
+
+export default function CarePlanConditionSelector() {
+
+    const { patientConditions, setCarePlanConditions } = useEnrollment()
+
+    const records = patientConditions?.map((condition) => {
+        return {
+            value: condition.id || "no-id",
+            label: condition.code?.text || "no-text"
+        }
+    })
+
+    if (!records) <>No Conditions selected</>
```

Comment:
@jorritspee do we want i18n support? (multi-language, in case we want to support Dutch) -> create issue
It's all in English now.

Reply:
https://linear.app/zorg-bij-jou/issue/INT-225/orca-or-support-i18n